website/demos/ScrollToCell.tsx

```diff
@@ -0,0 +1,82 @@
+import { useRef, useState } from 'react';
+
+import DataGrid from '../../src';
+import type { Column, FormatterProps, DataGridHandle } from '../../src';
+import type { Props } from './types';
+
+type Row = number;
+const rows: readonly Row[] = [...Array(200).keys()];
+const columns: Column<Row>[] = [];
+for (let i = 0; i < 200; i++) {
+  const key = String(i);
+  columns.push({
+    key,
+    name: key,
+    width: 120,
+    frozen: i < 5,
+    resizable: true,
+    formatter: cellFormatter
+  });
+}
+
+export default function ScrollToCell({ direction }: Props) {
+  const [idx, setIdx] = useState<number | undefined>(10);
+  const [rowIdx, setRowIdx] = useState<number | undefined>(10);
+  const gridRef = useRef<DataGridHandle>(null);
+
+  function scrollToCell() {
+    gridRef.current!.scrollToCell({ idx, rowIdx });
+  }
+
+  return (
+    <>
+      <div style={{ marginBlockEnd: 5 }}>
+        <label>
+          <span style={{ marginInlineEnd: 5 }}>Column index: </span>
+          <input
+            style={{ inlineSize: 50 }}
+            type="number"
+            value={idx}
```

Comment:
We're getting React warnings when it becomes undefined
```suggestion
            value={idx ?? ''}
```